src/matter/cluster/GeneralCommissioningCluster.ts

```diff
@@ -26,37 +26,52 @@ const CommissioningErrorT = { tlvType: TlvType.UnsignedInt } as Template<Commiss
 
 const BasicCommissioningInfoT = ObjectT({
     failSafeExpiryLengthSeconds: Field(0, UnsignedIntT),
+    maxCumulativeFailsafeSeconds: Field(1, UnsignedIntT),
 });
 
-const SuccessFailureReponseT = ObjectT({
+const CommissioningSuccessFailureResponseT = ObjectT({
     errorCode: Field(0, CommissioningErrorT),
     debugText: Field(1, StringT),
 });
-export type SuccessFailureReponse = JsType<typeof SuccessFailureReponseT>;
+export type CommissioningSuccessFailureResponse = JsType<typeof CommissioningSuccessFailureResponseT>;
 
 const ArmFailSafeRequestT = ObjectT({
     expiryLengthSeconds: Field(0, UnsignedIntT),
-    breadcrumbStep: Field(1, UnsignedIntT),
+    breadcrumbStep: Field(1, UnsignedLongT),
 });
 
 const SetRegulatoryConfigRequestT = ObjectT({
     config: Field(0, RegulatoryLocationTypeT),
     countryCode: Field(1, StringT),
-    breadcrumbStep: Field(2, UnsignedIntT),
+    breadcrumbStep: Field(2, UnsignedLongT),
 });
 
+/**
+ * This cluster is used to manage global aspects of the Commissioning flow.
+ */
 export const GeneralCommissioningCluster = Cluster(
     0x30,
     "General Commissioning",
     {
-        breadcrumb: Attribute(0, UnsignedIntT),
+        breadcrumb: WritableAttribute(0, UnsignedLongT, BigInt(0)), /* readAcl: view, writeAcl: administer */
```

Comment:
I think readAcl: view is the default so probably no need to specify it here.
I haven't see yet attributes with a higher read privilege, but there are probably some.

Reply:
Ok, I remove then